test/test_gpio.c

```diff
@@ -117,22 +132,25 @@ void test_invalid_address_gpio_set_pin_state(void) {
 }
 
 // Test that an invalid state is caught.
-void test_invalid_state_gpio_set_pin_state(void) {
+void test_gpio_invalid_state_set_pin_state(void) {
   TEST_ASSERT_TRUE(gpio_init());
   // A port that should be valid on all configurations.
-  GPIOAddress address = {0, 0};
+  GPIOAddress address = { 0, 0 };
```

Comment:
Not a huge deal, but I noticed that you've been using designated initializers, so you should probably stay consistent :)